alerts/AggregatedAPIErrors.md

```diff
@@ -0,0 +1,65 @@
+
+##### Check the APIs status checks are on True
+
+Currently, there are three aggregated APIs in an OpenShift Cluster. The API on the `openshift-apiserver` namespace, the prometheus-adapter on the namespace `openshift-monitoring`, and the package-server service in the `openshift-operator-lifecycle-manager` namespace.
+
+To get a list of the aggregated APIs and the `APIGroup` to whom they answer requests use the following command:
+
+```shell
+$ oc get apiservice -n openshift-apiserver | grep openshift-
+v1.apps.openshift.io                            openshift-apiserver/api                                      True        340d
+v1.authorization.openshift.io                   openshift-apiserver/api                                      True        340d
+v1.build.openshift.io                           openshift-apiserver/api                                      True        340d
+v1.image.openshift.io                           openshift-apiserver/api                                      True        340d
+v1.oauth.openshift.io                           openshift-apiserver/api                                      True        340d
+v1.packages.operators.coreos.com                openshift-operator-lifecycle-manager/packageserver-service   True        340d
+v1.project.openshift.io                         openshift-apiserver/api                                      True        340d
+v1.quota.openshift.io                           openshift-apiserver/api                                      True        340d
+v1.route.openshift.io                           openshift-apiserver/api                                      True        340d
+v1.security.openshift.io                        openshift-apiserver/api                                      True        340d
+v1.template.openshift.io                        openshift-apiserver/api                                      True        340d
+v1.user.openshift.io                            openshift-apiserver/api                                      True        340d
+v1beta1.metrics.k8s.io                          openshift-monitoring/prometheus-adapter                      True        340d
+```
+Here we want the availability status to be `True`. A `False` means that requests for that API service, API server pods, or resources belonging to that apiGroup failed many times during the last minutes.
+
+##### Check the `apiserver` pods
+```shell
+$ oc get pods -n openshift-apiserver
+NAME                        READY   STATUS    RESTARTS   AGE
+apiserver-c74d6bd79-pkmw4   1/1     Running   0          41d
+apiserver-c74d6bd79-t4l9d   1/1     Running   0          41d
+apiserver-c74d6bd79-z6qxx   1/1     Running   0          41d
+```
+If their status is not `Running` check the logs for more details. 
+
+##### Check the authentication certificates of the aggregated API
+Make sure the certificates are up to date and still valid. Use:
+```shell
+$ oc get configmaps -n kube-system extension-apiserver-authentication
```

Comment:
nit: `shell` is for shell snippets, e.g. things you could drop in a `whatever.sh` script behind a `#!/bin/sh` and run.  But here you've included a `$ ` prompt.  You should either drop the prompt:

```shell
oc get configmaps -n kube-system extension-apiserver-authentication
```

or use `console`:

```console
$ oc get configmaps -n kube-system extension-apiserver-authentication
```